website/docs/foundations/colors/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,94 @@
+
+- Strong
+- Primary
+- Faint
+- Action (not to be used outside of context)
+- Disabled (not to be used outside of context)
+
+### Foreground colors
+
+Foreground colors are used for elements such as text, links, and icons.
+
+Some common examples of semantic foreground colors include:
+
+- `Foreground/Strong` for headings and secondary links
+- `Foreground/Primary` for body text 
+- `Foreground/Faint` for less prominent text and UI elements.
+- `Foreground/Action` for primary call to action such as links
+
+![Samples of text with their color tokens labelled](/assets/foundations/color/colors-foreground-examples.png)
+
+Use status foreground colors to help contextualize responses from user actions or to indicate status within a UI. These colors should be used sparingly and within the context of specific components like [Alerts](/components/alert#color), [Toasts](/components/toast#color) or [Badges](/components/badge#color).
+
+![Sample positive, warning, and error status badges with their associated foreground color tokens labelled.](/assets/foundations/color/colors-status-examples.png)
+
+### Border colors
+
+Some common examples of border colors include:
+
+- `Border/Strong` for secondary Button border
+- `Border/Primary` for Card border or divider
+- `Border/{Status color}` for Alert borders
+
+![Sample borders applied on a button, card and alert with the border color tokens labelled](/assets/foundations/color/colors-border-examples.png)
+
+### Surface colors
+
+Use surface colors for the background (or surface) of a component or container. 
+
+Some common examples of surface colors include:
+
+- `Surface/Strong` for the neutral Badge.
+- `Surface/Primary` for component containers.
+- `Surface/Faint` for the secondary Button.
+- `Surface/{Status color}` for Alert background.
+
+![Sample surface colors being applied to a badge, card, button and alert with the surface color token labelled.](/assets/foundations/color/colors-surface-examples.png)
+
+### Page colors
+
+Page colors are used for page backgrounds. HDS components do not use these tokens, however, we recommend `Page/Primary` as the primary background color and `Page/Faint` as a means to create a secondary level on the page for highlighting information, if necessary.
+
+## Accessible color combinations
+
+We intend to be conformant with WCAG 2.2 Level AA requirements. In terms of color contrast, this means a luminosity ratio of 4.5:1 for normal sized text, and 3:1 for large text (commonly 22px). Further details are outlined on [WCAG’s understanding of Contrast (Minimum)](https://www.w3.org/WAI/WCAG22/Understanding/contrast-minimum.html). By default, semantic color tokens provide accessible color combinations out of the box with their associated naming conventions, while using palette colors requires manual validation, especially if you plan to mix and match. As an example, if a color has a semantic status name in it, then other associated status colors will be accessible. 
```

Comment:
For this part: "As an example, if a color has a semantic status name in it" does this mean that if the color **_name_** includes a semantic status...?

Reply:
This would be referencing things like `Foreground/Success` or `Foreground/Success-On Surface` would pair with `Surface/Success`.